<?xml version="1.0" encoding="UTF-8"?>
<!-- 
//
// Copyright (c) 2021 The University of Aizu
// Open source under the BSD License.
//
 -->
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>0</y>
      <width>1041</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="Rendering" name="modelRendering">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>340</y>
        <width>491</width>
        <height>276</height>
       </rect>
      </property>
     </widget>
     <widget class="ShowImage" name="imageViewer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>494</width>
        <height>276</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>1021</width>
        <height>159</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Pixel information</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="pixelInfoList">
           <item>
            <property name="text">
             <string>coordinate of pixel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>value</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>...</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Data export</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>x </string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLineEdit" name="coordinate_x"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Coordinate of pointer</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLineEdit" name="coordinate_y"/>
         </item>
         <item row="7" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Select value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>Digital Number</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Temperature</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Select temparature correction method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Nothing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Correction formula</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bilinear</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Select Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modelSelect">
           <item>
            <property name="text">
             <string>Itokawa</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ryugu</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>351</width>
        <height>184</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QPushButton" name="loadDataButton">
         <property name="text">
          <string>Load data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Loaded data list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="loadedDataList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="exportWindowButton">
           <property name="text">
            <string>Export to window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Search similar image</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>351</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Window list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="windowList"/>
       </item>
       <item>
        <widget class="QPushButton" name="tilingWindowButton">
         <property name="text">
          <string>Tiling window</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSelect_TIR_directory">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Select TIR directory</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionChange_DB_info">
   <property name="text">
    <string>&amp;Change DB info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ShowImage</class>
   <extends>QOpenGLWidget</extends>
   <header>showimage.h</header>
  </customwidget>
  <customwidget>
   <class>Rendering</class>
   <extends>QOpenGLWidget</extends>
   <header>rendering.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
